shared/naturalcrit/markdown.js

```diff
@@ -333,35 +334,6 @@ const mustacheInjectBlock = {
 	}
 };
 
-const superSubScripts = {
-	name  : 'superSubScript',
```

Comment:
I see the package is named "subSuper" but here it is superSub. Let's make it consistent either way.

Reply:
Whoops. That's an error. Should be subSuper.

Reply:
Where I perform the import is it named `MarkedSubSuperText`

Reply:
I have renamed the block type in the module for consistency.